docs/user-guide/user-roadmap-zowe-cli copy.md

```diff
@@ -0,0 +1,161 @@
+# Installing Zowe CLI overview
+
+The resources here provide information about various Zowe CLI topics, such as learning basic skills, installation, developing, and troubleshooting.
+
+:::tip
+
+To identify the resources most relevant for you, use the following definitions of Zowe CLI skill levels.
+
+- **Beginner:** You're starting out and want to learn the fundamentals.
+- **Intermediate:** You have some experience but want to learn more in-depth skills. 
+- **Advanced:** You have lots of experience and are looking to learn about specialized topics.
+
+:::
+
+## Fundamentals
+
+> Zowe skill level: Beginner
+
+- [**Zowe CLI overview**](../getting-started/overview.md#zowe-cli)
+
+   New to Zowe CLI? This overview topic provides an introduction to Zowe CLI.
+
+- [**Architecture**](../getting-started/zowe-architecture.md#zowe-architecture)
+
+   Review the Zowe architecture to understand how Zowe CLI works in the Zowe framework.
+
+- [**Zowe CLI FAQs**](../getting-started/zowe_faq.md#zowe-cli-faq)
+
+   If you have a question, check out the FAQ, which answers the most commonly asked questions about Zowe CLI.
+
+## Quick start
+
+> Zowe skill level: Beginner
+
+- [**Zowe CLI quick start**](../getting-started/cli-getting-started.md)
+
+   Get started with Zowe CLI quickly and easily.
+
+- [**Blog: Getting Started with Zowe CLI**](https://medium.com/zowe/getting-started-with-zowe-cli-7a29420c6be7)
+
+   This blog explains how to get started with Zowe CLI quickly.
+
+## Installing
+
+> Zowe skill level: Beginner
+
+- [**System requirements**](../user-guide/systemrequirements-cli.md)
+
+   Review this topic to ensure that your system meets the requirements for installing Zowe CLI. 
+
+- [**Installing Zowe CLI**](../user-guide/cli-installcli.md)
+
+   Follow the steps to install a new release of Zowe CLI.
+
+## Configuring and updating
+
+> Zowe skill level: Intermediate
+
+- [**Configuring Zowe CLI environment variables**](../user-guide/cli-configuringcli-ev.md)
+
+   Explains how to configure Zowe CLI environment variables, such as changing log levels, setting the home directory location, and daemon mode.
+
+- [**Updating Zowe CLI**](../user-guide/cli-updatingcli.md)
+
+   Learn how to update Zowe CLI to a more recent version using different methods. 
+
+## Using Zowe CLI and plug-ins
+
+> Zowe skill level: Intermediate
+
+- [**Using Zowe CLI**](../user-guide/cli-using-usingcli.md)
+
+   Learn about how to use Zowe CLI, including connecting to the mainframe, managing profiles, integrating with API Mediation Layer, and more.
+
+- [**Zowe CLI extensions and plug-ins**](../user-guide/cli-extending.md)
+
+   This information guides you to explore the extensions and plug-ins available to Zowe CLI and install plug-ins to extend the capabilities of Zowe CLI. Plug-ins add functionality to the product in the form of new command groups, actions, objects, and options.
+
+- **Docs: Zowe CLI command reference guide**
```

Comment:
I see we moved the `appendix/zowe-cli-command-reference` topic to the roadmap 🥳 

Curious if we should pick one of the 3 `href`-s below and make this bullet point a hyperlink.

I'm just asking since it's one of the very few that isn't a link 😋

Reply:
@zFernand0 -- this was address in the non-copy version of this file. again, my apologies for the confusion.